cmd/docgen/rule.go

```diff
@@ -62,5 +62,14 @@ func trivyRules() Rules {
 			SubCategory: "Cryptography",
 			Enabled:     true,
 		},
+		{
```

Comment:
I asked the Claudias for feedback on this and we got to slightly different strings (https://codacy.atlassian.net/browse/TAROT-2384)